app/views/components/docs/contents-list.yml

```diff
@@ -0,0 +1,100 @@
+name: Contents list
+description: Lists a page’s contents with links pointing to headings within the document
+body: |
+  Pass a list of contents each with an `href` and `text`. The `href` should point at the ID of a heading within the page.
+
+  Supports nesting contents one level deep, currently only used by specialist documents. When nesting the top level list items
+  display in bold.
+
+  `format_numbers` option will pull out numbers in the link text to render them as though they were the list style type. Applies to
+  numbers at the start of text, with or without a decimal. See the [format complex numbers fixture](/component-guide/contents-list/formats_complex_numbers) for details.
+accessibility_criteria: |
+  The contents links must:
+
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - indicate when it has focus
+fixtures:
+  default:
+    contents:
+      - href: "#first-thing"
+        text: First thing
+      - href: "#second-thing"
+        text: Second thing
+      - href: "#third-thing"
+        text: Third thing
+  long_text:
+    contents:
+      - href: "#first-thing"
+        text: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      - href: "#second-thing"
+        text: Another pretty long contents list entry, not as long as the first, but still a little.
+      - href: "#third-thing"
+        text: Third thing
+  nested_contents_lists:
+    contents:
+      - href: "#first-thing"
+        text: First thing
+      - href: "#second-thing"
+        text: Second thing
+      - href: "#third-thing"
+        text: Third thing
+        items:
+          - href: "#sub-third-thing"
+            text: Sub third thing
+          - href: "#another-third-thing"
+            text: Another third thing
+      - href: "#fourth-thing"
+        text: Fourth thing
+  formats_numbers:
+    format_numbers: true
+    contents:
+      - href: "#first-thing"
+        text: 1. First thing
+      - href: "#second-thing"
+        text: 2. Second thing
+      - href: "#third-thing"
+        text: 3. Third thing
+  formats_complex_numbers:
+    format_numbers: true
+    contents:
+      - href: "#"
```

Comment:
Is there a reason that all the other fixtures have a full href value like #first-thing but the ones in this one are just #?